probe-rs/src/core/mod.rs

```diff
@@ -493,6 +493,9 @@ pub trait CoreInterface: MemoryInterface {
     /// Returns `true` if hwardware breakpoints are enabled, `false` otherwise.
     fn hw_breakpoints_enabled(&self) -> bool;
 
+    /// Configure the debug module to ensure software breakpoints will enter Debug Mode.
```

Comment:
I think `debug_module` is a RISCV specific term, maybe we can say `Configure the core ..`or something similar?

Reply:
Thanks :)